sanic/app.py

```diff
@@ -835,6 +835,11 @@ def run(
         access_log=True,
         **kwargs
     ):
+        try:
+            assert "loop" not in kwargs
+        except AssertionError:
+            raise TypeError("loop is not a valid argument. For asynchronous support, check https://sanic.readthedocs.io/en/latest/sanic/deploying.html#asynchronous-support")
```

Comment:
How about:

"loop is not a valid argument. To use an existing loop, change to create_server().\nSee more: https://sanic.readthedocs.io/en/latest/sanic/deploying.html#asynchronous-support"